Muehle_Conclusion.ipynb

```diff
@@ -62,782 +71,125 @@
    "outputs": [],
    "metadata": {}
   },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Im folgenden Programmcode werden, abhängig von den Variablen d und wdh, d*wdh-viele Spiele durchgerechnet und die die durchnittlichen Zugzeiten für die Algorithmen bestimmt. Hierzu wird nach jedem Spiel für Alha-Beta und Minimax die Gesamtzeit des jeweiligen Algorithmus durch die  Anzahl der Züge des Algorithmus geteilt. Somit entsteht die durschnittliche Zugzeit für den Algorithmus. \r\n",
+    "* `d`   ist die Tiefe, bis zu der Minimax mit Alpha-Beta verglichen werden soll\r\n",
+    "* `wdh` ist die Anzahl der Wiederholungen einer Tiefenberechnung die für die Bildunge des Durchschnitts genutzt werden\r\n",
+    "\r\n",
+    "Bei mehrfachem Ausführen fällt hier auf, dass die Werte für d∈{1,2} stark schwanken und somit nicht besonders aussagekräftig sind. "
+   ],
+   "metadata": {}
+  },
   {
    "cell_type": "code",
-   "execution_count": 4,
+   "execution_count": 10,
    "source": [
     "import random \r\n",
     "import pandas as pd\r\n",
```

Comment:
pandas verwendet standardmäßig matplotlib. Deswegen muss das dann glaube ich auch noch in requirements.txt geschrieben werden